### DISCUSSION TITLE
The C code is longer than Ruby/Batchfile but it's included in "Other" category.

### DISCUSSION THREAD

**Discussion by Yilmaz4** [github-linguist/linguist#5429] [Classification] [answered]
I maked a simple Hello World program in all possible languages, I created the Ruby code earlier and C code later. But even C code is longer than Ruby code, it's still included in Other category and I want visitors to see "C" language in Languages section instead of Batchfile. Is this a bug or something I don't know?

This is [my repository](https://github.com/Yilmaz4/Hello-World/)

**Comment by lildude** [comment, marked as answer]
GitHub will only show at most 6 languages in the side bar, and it'll show the six most popular by bytes of code. The rest will be grouped into "Other". You're seeing Batchfile instead of C because you have more bytes of Batchfile code than C:


    C# 3.91 KB
    Visual Basic .NET 3.63 KB
    Java 3.21 KB
    Python 2.98 KB
    Ruby 2.68 KB
    Batchfile 2.54 KB
    C++ 2.24 KB
    C 2.01 KB
    F# 64 Bytes
    Shell 29 Bytes
    BASIC 23 Bytes

If you don't want a language to show up, you can use an override to tell Linguist to ignore it.

As this isn't a bug and is expected behaviour, I'm moving this to Discussions.

**Reply by Yilmaz4** [reply]
![Untitled](https://user-images.githubusercontent.com/77583632/122021156-12f5fc80-cdce-11eb-8adc-338410a17927.png)
But why most of languages including C is not showing here?

**Reply by lildude** [reply]
That's Search functionality which Linguist has no control over, however from what I can see, this is limited to showing only 10 languages, but I can't see which languages it chooses to show; it seems to change slightly with each page refresh too.

If you really want to know, I'd suggest you contact GitHub Support.
